libraries/default.rb

```diff
@@ -0,0 +1,68 @@
+def collect_github_repositories(github_token, org)
+  if ::ObjectSpace.const_defined?('Chef')
+    # Chef exists! Lets make sure Octokit is installed.
+    chef_gem 'octokit'
+  end
```

Comment:
The functions in the library have the potential to be called by recipes other than the cookbook_uploader recipe.  These functions depend on Octokit, so we need to enforce that we have it where the functions are.  If requiring Octokit twice is not ideal, then we should either remove the Octokit installation from the cookbook_uploader recipe or move the functions to the cookbook_uploader recipe.